src/components/panel/panel.spec.js

```diff
@@ -1475,6 +1696,29 @@ describe('$mdPanel', function() {
   }
 
   /**
+   * Attaches the panel. If a config value is passed, creates a new panelRef
+   * using $mdPanel.open(config); Otherwise, called open on the panelRef,
```

Comment:
Please update this description. I also lean away from creating a panel here. You can do that in a test that uses this function, since you don't need a config to create a panel.

If you are allowing the create panel here, you'll need to support both having and not having a config. (Which means you'll need to keep track somewhere in this test whether a panel's been created or not. Since `open` also creates, you don't have an easy way to do that check.)